src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -31,10 +31,12 @@ export interface ConnectorTransaction {
   /**
    * Settle date in UNIX milliseconds
    */
-  settleDate: number;
+  settleDate?: number;
+  creationDate?: number;
   totalAmount: number;
   displayAmount?: [number, ACCOUNT_CURRENCIES];
   type: "received" | "sent";
+  state?: string;
```

Comment:
can you use typing here for the possible states

Reply:
resolved